test/syscalls/linux/proc_net.cc

```diff
@@ -54,6 +54,44 @@ TEST(ProcSysNetIpv4Sack, CanReadAndWrite) {
   EXPECT_EQ(buf, to_write);
 }
 
+TEST(ProcSysNetIpv4IpForward, Exists) {
+  auto fd =
+      ASSERT_NO_ERRNO_AND_VALUE(Open("/proc/sys/net/ipv4/ip_forward", O_RDWR));
+
+  fd.reset();
```

Comment:
This shouldn't be necessary. fd's destructor should close the FD for you.

Reply:
fixed